docs/content/en/docs/install/install.md

```diff
@@ -109,10 +109,9 @@ For more information,see
 
 ## Use manifests
 
-All versions of the Lifecycle Toolkit can be installed using manifests,
+Versions v0.10.0 and earlier of the Lifecycle Toolkit can be installed using manifests,
 although we recommend that you use Helm Charts
```

Comment:
```suggestion
although we recommend that you use Helm Charts for version 0.7.0 and later
```
It doesn't make sense to recommend that they use Helm Charts for versions
that did not support them ;-)

Reply:
we have helm charts since KLT 0.7.0 though :)